CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ Note that this project **does not** adhere to [Semantic Versioning](http://semve
 ### Added
 
 - We added import support for CFF files. [#7945](https://github.com/JabRef/jabref/issues/7945)
+- We added support to use Lucence search syntax also for the searchin the library or in dynamic groups.
```

Comment:
```suggestion
- We added support to use Lucence search syntax also for the search in the library or in dynamic groups.
```

Reply:
"Lucene search syntax" is also not very user friendly description, most of our users shouldn't know what lucene is.

Reply:
Unified search syntax? Needs documentation and migration changes guide. eg how to search for fields and regex stuff etc